descriptors/ModuleDescriptor-template.json

```diff
@@ -405,6 +405,9 @@
       },
       { "name": "MAX_QUERY_SIZE",
         "value": "1250000"
+      },
+      { "name": "mod.fqm-manager.entity-type-cache-timeout-seconds",
+        "value": "300"
```

Comment:
This seems high enough to make an impact, but still low enough to not make updating entity types frustrating.

Reply:
The cache still gets nuked on update, yeah? If we want 300 to be the default I'd sync the readme/application.properties, too

Reply:
Yep, anything that restarts the JVM (or even that nukes the `EntityTypeRepository` instance) would clear the cache. I think 3600 (1 hour) feels like a reasonable default for production. That's a terrible default for dev envs, though. I was thinking 300 strikes a nice balance where it's okay for dev without being awful for production (still not great for prod, but at least it's not awful)

Reply:
I went ahead and added a paragraph to the readme to make it clear that the value in the module descriptor is different from the default, along with my reasoning. I also added some simple recommendations for production (basically: higher timeouts = good. 1 day or 1 week would be solid choices)